apps/web/screens/UniversitySearch/UniversitySearch.tsx

```diff
@@ -0,0 +1,1253 @@
+                setSearchTerm(e.target.value)
+                searchTermHasBeenInitialized.current = true
+                setQuery(e.target.value)
+              }}
+            />
+            <ContentBlock>
+              <Box paddingTop={2}>
+                <Inline space={[1, 2]}>
+                  <Tag
+                    onClick={() => resetFilteredList()}
+                    customClassName={
+                      Object.keys(filters).filter((x) => x.length > 0).length >
+                      0
+                        ? styles.tagActive
+                        : styles.tagNotActive
+                    }
+                  >
+                    {n('showAll', 'Sýna allt')}
+                  </Tag>
+                  {filterOptions.map((option) => {
+                    if (
+                      option.field === 'degreeType' ||
+                      option.field === 'universityId'
+                    ) {
+                      return option.options.map((item) => {
+                        if (item !== 'OTHER') {
+                          return (
+                            <Tag
+                              onClick={() => handleFilters('degreeType', item)}
+                              customClassName={
+                                filters.degreeType.includes(item)
+                                  ? styles.tagActive
+                                  : styles.tagNotActive
+                              }
+                            >
+                              {n(
+                                option.field === 'universityId'
+                                  ? universities.filter((x) => x.id === item)[0]
+                                      .title
+                                  : item,
+                                option.field === 'universityId'
+                                  ? universities.filter((x) => x.id === item)[0]
+                                      .title
+                                  : item,
+                              )}
+                            </Tag>
+                          )
+                        }
+                      })
+                    }
+                  })}
+                </Inline>
+              </Box>
+            </ContentBlock>
+
+            <Hidden above="md">
+              <Box width="full" marginTop={2}>
+                <Filter
+                  resultCount={filteredResults?.length ?? 0}
+                  variant={'dialog'}
+                  labelClear={n('clear', 'Hreinsa')}
+                  labelClearAll={n('clearAllFilters', 'Hreinsa allar síur')}
+                  labelOpen={n('open', 'Opna')}
+                  labelClose={n('close', 'Loka')}
+                  labelResult={n('showResults', 'Skoða niðurstöður')}
+                  labelTitle={n('filterResults', 'Sía niðurstöður')}
+                  onFilterClear={() => {
+                    //TODO fix the bug that always returns initialFilter with new items!!
```

Comment:
is it something that can be fixed now?

Reply:
This isn't going live and there is a lot of pressure to get this into testing, so I am working on this now but hoping it won't stop the PR from going through

Reply:
Aahh, i see, classic hehe :D 

However, this is that big of a PR, that doesnt seem to be complete, 
could this not be tested as a feature branch?

Reply:
Vésteinn wants this out on dev and so we are trying to get this out under a feature flag (contentful ATM). We don't know how to do this another way and still have all these new features work.

Reply:
Is there anything stopping you from adding the `deploy-feature` label on this PR?